<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BehaviourSettingsPage</class>
 <widget class="QWidget" name="BehaviourSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Behavior</string>
  </property>
  <layout class="QVBoxLayout" name="layout_showtrayicon">
   <item>
    <widget class="QCheckBox" name="checkbox_showtrayicon">
     <property name="text">
      <string>Show system tray icon</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_scrolltrayicon">
     <property name="text">
      <string>Scroll over icon to change track</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_keeprunning">
     <property name="text">
      <string>Keep running in the background when the window is closed</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_startupgroup">
     <property name="title">
      <string>When Strawberry starts</string>
     </property>
     <layout class="QVBoxLayout" name="layout_mainwindow">
      <item>
       <widget class="QRadioButton" name="radiobutton_alwaysshow">
        <property name="text">
         <string>Always show the main window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiobutton_alwayshide">
        <property name="text">
         <string>Always hide the main window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiobutton_remember">
        <property name="text">
         <string>Remember from last time</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer_1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_resumeplayback">
        <property name="text">
         <string>Resume playback on start</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_seekstep">
     <property name="title">
      <string>Seeking using a keyboard shortcut or mouse wheel</string>
     </property>
     <layout class="QHBoxLayout" name="layout_seekstep">
      <item>
       <widget class="QLabel" name="label_seekstep">
        <property name="text">
         <string>Time step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinbox_seekstepsec">
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
